test/test_sparse.py

```diff
@@ -2190,9 +2190,8 @@ def is_integral(dtype):
 
             # test autograd
             x = sparse_tensor.clone()
-            y = sparse_tensor.log1p()
-            with self.assertRaisesRegex(RuntimeError, "log1p of a sparse tensor is made to be non-differentiable"):
-                y.backward(x)
+            #y = sparse_tensor.log1p()
```

Comment:
Should we leave this?

Reply:
Looking a bit closer, the autograd stuff should be covered by `TestSparseUnaryUfuncs` but there was a skip that needed to be removed from the opinfo. 

I have actually removed a good chunk of this test which is redundant, however some of the behaviors are not covered by the opinfo based test and I figured they should stay.